### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by ecorkevin**

### CODE REVIEW

**Comment by coreybutler** [comment]
Can you check this against the merge conflicts? I meant to include this before a bunch of changes, but I forgot.

File: src/net.js [hunk 1]
```diff
@@ -278,7 +278,7 @@ class Network {
        * The URL of the operation.
        * @param {string} rel
        * The type of operation. For example: `preconnect`.
-       * @param {boolean} [crossorigin]
+       * @param {boolean} [cor] crossorigin
```
Comment (coreybutler) [inline src/net.js]: The original form of this was correct (i.e. crossorigin). By changing this to "cor", the name of the attribute  in the documentation will be changed!

File: src/net.js [hunk 2]
```diff
@@ -652,7 +652,7 @@ class Network {
    * time. This helps the browser resolve DNS inquiries quickly.
    * @param {string} domain
    * The domain to resolve.
-   * @param {boolean} [crossorigin]
+   * @param {boolean} [cor] crossorigin
```
Comment (coreybutler) [inline src/net.js]: See prior comment (same issue).

File: src/net.js [hunk 3]
```diff
@@ -672,7 +672,7 @@ class Network {
    * a resource in response to a user interaction or NGN.BUS activity.
    * @param {string} url
    * The URL to preconnect to.
-   * @param {boolean} [crossorigin]
+   * @param {boolean} [cor] crossorigin
```
Comment (coreybutler) [inline src/net.js]: See prior comment (same issue).

File: src/net.js [hunk 4]
```diff
@@ -688,7 +688,7 @@ class Network {
    * Fetch a specific resource and cache it.
    * @param {string} url
    * URL of the resource to download and cache.
-   * @param {boolean} [crossorigin]
+   * @param {boolean} [cor] crossorigin
```
Comment (coreybutler) [inline src/net.js]: See prior comment (same issue).

File: src/net.js [hunk 5]
```diff
@@ -706,7 +706,7 @@ class Network {
    * as "needed ASAP". Otherwise, use #prefetch.
    * @param {string} url
    * URL of the resource to download and cache.
-   * @param {boolean} [crossorigin]
+   * @param {boolean} [cor] crossorigin
```
Comment (coreybutler) [inline src/net.js]: See prior comment (same issue).

File: src/net.js [hunk 6]
```diff
@@ -721,13 +721,13 @@ class Network {
...
    * opened in a hidden tab, then displayed when called. This is
-   * powerful, but should only be used when there is absolutely
+   * powerful, but should only be used when there is absolute
    * certainty that the prerendered page will be needed. Otherwise
    * all of the assets are loaded for no reason (i.e. uselessly
    * consuming bandwidth).
    * @param {string} url
    * URL of the page to download and cache.
-   * @param {boolean} [crossorigin]
+   * @param {boolean} [cor] crossorigin
```
Comment (coreybutler) [inline src/net.js]: See prior comment (same issue).

**Comment by coreybutler** [comment]
So this is fine from a merge conflict perspective, but there are some issues with the changes, specifically on the NGN.NET file. I added comments so you'd have a checklist to go through. Fix those and I'll merge.

File: src/net.js (hunk 1 above)
Comment (ecorkevin) [inline src/net.js]: I was under the impression that the name of the argument (cor) came first, and then the description. Perhaps the description should start on a new line instead? If it's "crossorigin" should that be reflected on line 287?

Comment (coreybutler) [inline src/net.js]: The name of the configuration property is `crossorigin`, not `core`. It does make sense to change it on line 287 as well, though it is not _technically_ required.

**Pull request merged**